opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -561,6 +561,12 @@ message ExponentialHistogramDataPoint {
 
   // max is the maximum value over (start_time, end_time].
   optional double max = 13;
+
+  // ZeroThreshold may be optionally set to convey the width of the
+  // zero region. When ZeroThreshold is unset or 0, this bucket stores values
```

Comment:
`this bucket` is mileading. I think you mean the `zero_count`?

Reply:
maybe `this bucket` -> `zero count bucket`?